dynamicisttoolkit-dev-env.yml

```diff
@@ -3,6 +3,7 @@ channels:
   - conda-forge
 # NOTE : The minimum versions correspond to those in Ubuntu 22.04 LTS.
 dependencies:
+  - python == 3.11
```

Comment:
We need to leave this out for the CI so that it can set python. We can switch from nose to pytest so that we can run on Python 3.12 in the future. See #44

Reply:
Alright.

Reply:
I fixed this in master.